### PR TITLE
Adding missing functions.

### PR DESCRIPTION
**Pull request by NiklasGustafsson**
amin(), amax(), complex(), and polar() were all missing. Now they're not.

### CODE REVIEW

**Comment by NiklasGustafsson** [comment]
Tests failing due to floating point tolerance error in allclose(). Looking into mitigation.

File: src/Native/LibTorchSharp/THSTensor.h [hunk 1]
```diff
@@ -220,6 +224,10 @@ EXPORT_API(Tensor) THSTensor_clamp_min(const Tensor input, const Scalar min);
 
 EXPORT_API(Tensor) THSTensor_clamp_min_(const Tensor input, const Scalar min);
 
+EXPORT_API(Tensor) THSTensor_clamp_min_(const Tensor input, const Scalar min);
```
Comment (dsyme) [inline src/Native/LibTorchSharp/THSTensor.h:227]: Looks like a duplicate here?

File: src/TorchSharp/Tensor/TorchTensor.cs [hunk 2]
```diff
@@ -1361,6 +1361,46 @@ public TorchTensor all(long dimension, bool keepDim = false)
...
+        [DllImport("LibTorchSharp")]
+        static extern IntPtr THSTensor_amin(IntPtr tensor, IntPtr dim, int dim_len, bool keep_dim);
+
+        /// <summary>
+        /// Returns the minimum value of each slice of the input tensor in the given dimension(s) dim.
+        /// </summary>
+        /// <param name="dims">The dimension or dimensions to reduce.</param>
+        /// <param name="keepDim">Whether the output tensor has dim retained or not.</param>
+        /// <returns></returns>
+        public TorchTensor main(long[] dims, bool keepDim = false)
```
Comment (dsyme) [inline src/TorchSharp/Tensor/TorchTensor.cs:1393]: typo - `amin` not `main`

**Review by dsyme — COMMENTED** [review]
Couple of small changes needed. Don't wait for a re-review :-)

File: src/TorchSharp/Tensor/TorchTensor.cs (hunk 2 above)
Comment (dsyme) [reply src/TorchSharp/Tensor/TorchTensor.cs:1393]: Indicates a missing test?

Comment (NiklasGustafsson) [reply src/TorchSharp/Tensor/TorchTensor.cs:1393]: Yup. 'amin' is the right spelling.

**Pull request merged**